testsuite/cluster-test/src/experiments/performance_benchmark.rs

```diff
@@ -101,15 +118,25 @@ impl Experiment for PerformanceBenchmark {
         let buffer = Duration::from_secs(60);
         let window = self.duration + buffer * 2;
         let emit_job_request = if context.emit_to_validator {
-            EmitJobRequest::for_instances(
```

Comment:
Quick nit - you can avoid too much code duplication by splitting this nested if+matcg into to:
```
let instances = if context.emit_to_validator {self.up_validators.clone()} else {...};
match self.tps {...}
```